freqtrade/exchange/exchange.py

```diff
@@ -2084,19 +2270,22 @@ def refresh_latest_ohlcv(self, pair_list: ListPairsWithTimeframes, *,
         Only used in the dataprovider.refresh() method.
         :param pair_list: List of 2 element tuples containing pair, interval to refresh
         :param since_ms: time since when to download, in milliseconds
-        :param cache: Assign result to _klines. Usefull for one-off downloads like for pairlists
+        :param cache: Assign result to _trades. Usefull for one-off downloads like for pairlists
         :param drop_incomplete: Control candle dropping.
             Specifying None defaults to _ohlcv_partial_candle
         :return: Dict of [{(pair, timeframe): Dataframe}]
         """
-        logger.debug("Refreshing candle (OHLCV) data for %d pairs", len(pair_list))
+        logger.debug(
+            "Refreshing candle (OHLCV) data for %d pairs", len(pair_list))
 
         # Gather coroutines to run
-        input_coroutines, cached_pairs = self._build_ohlcv_dl_jobs(pair_list, since_ms, cache)
+        input_coroutines, cached_pairs = self._build_ohlcv_dl_jobs(
+            pair_list, since_ms, cache)
 
         results_df = {}
         # Chunk requests into batches of 100 to avoid overwelming ccxt Throttling
         for input_coro in chunks(input_coroutines, 100):
+
             async def gather_stuff():
                 return await asyncio.gather(*input_coro, return_exceptions=True)
 
```

Comment:
it's already a pretty large PR - if that then includes unrelated formatting changes, it becomes very difficult to properly review / identify the ACTUAL, functional changes.
an example in the highlighted code-section:

* the comment change is wrong (the method is called `refresh_latest_ohlcv`)
* all other changes are whitespace / line only changes